async-http.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
 	spec.add_dependency "protocol-http", "~> 0.24.0"
 	spec.add_dependency "protocol-http1", "~> 0.15.0"
 	spec.add_dependency "protocol-http2", "~> 0.15.0"
-	spec.add_dependency "traces", ">= 0.8.0"
+	spec.add_dependency "traces", ">= 0.10.0"
```

Comment:
Adding this dependency now mean async-http needs at least Ruby 2.7 to run but that is not reflected in the gemspec

Noticed this because Sinatra CI failed on Ruby 2.6: https://github.com/sinatra/sinatra/actions/runs/5263567184/jobs/9631906714#step:5:15

Reply:
Ah thanks for the report. I have a general policy of not supporting EOL Rubies. However, we can probably drop 2.7 as a requirement for traces in the next release because we won't need to use `...` any more. Hopefully I'll be able to do the next release in about a week.

Reply:
Sounds good, I've worked around the problem in Sinatra CI but I imagine it could be more difficult to do that in other projects (applications). 

Just curious, should you be specifying the supported Ruby version in the gemspec for your gems? Then users of old Ruby versions can just keep using old versions of the gems (bundler should just pick the correct version for them).

Reply:
I usually try to do so, I'll probably need to automate it, I automate a lot of maintenance but don't always remember to fix the minimum version.